ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
+
+from helpers import Singleton, setup_logger
+
+DBusGMainLoop(set_as_default=True)  # has to be called first
```

Comment:
```
threads_init() 
Initialize threads in dbus-glib, if this has not already been done.
This must be called before creating a second thread in a program that uses this module.
```
[(from here)](https://dbus.freedesktop.org/doc/dbus-python/api/dbus.mainloop.glib-module.html)

Reply:
I wonder if this applies to us. If so, that's a big problem since many threads are created before app import even starts